Reference.md

```diff
@@ -3108,26 +3116,29 @@ There are a lot of different moves that involve cluing trash. [Here is a handy f
-- Just like a *Bluff*, a *Charm* can only be performed on the very next player.
+- *Charm* is similar to *Ejection* and *Discharge*, except the player plays their *Fourth Finesse Position*.
+- *Charms* arise when playing *First*, *Second*, **or** *Third Finesse Position* would cause *Information Desynchronization* about the initial clue.
+- Unlike *Ejection* and *Discharge*, the original *Signal* of the clue is maintained in a *Charm*. Instead, this blind-play indicates there was unique significance in the specific clue chosen.
+- Just like a *Bluff*, a *Charm* can only be performed on the very next player.
 
 ### The Safety Charm (Declined 5 Pull)
 
 - This convention only applies in the *Low Score Phase*.
 - If a card could be freely gotten by initiating a *5 Pull*, it would be very strange if the *5 Pull* was declined and the card was directly touched as a 1-for-1.
 - When this occurs, this player is trying to communicate something deeper - a *Charm* on the very next player.
-- Normally, when an card is blind-played after an innocent-looking *Play Clue*, the clue receiver knows to not play that card anymore. In this situation, since the *Fourth Finesse Position* card was played instead of the *First*, *Second*, or *Third*, the clue receiver will know that their card is still playable **and** that the unclued card directly to its left must be a 5.
+- Normally, when a card is blind-played after an innocent-looking *Play Clue*, the clue receiver knows to not play that card anymore. In this situation, since the *Fourth Finesse Position* card was played instead of the *First*, *Second*, or *Third*, the clue receiver will know that their card is still playable **and** that the unclued card directly to its left must be a 5.
 - This untouched 5 is now globally known; everyone on the team should treat it as a *Chop Moved* card.
+- This can be understood as a full-circle *Signal Shift*, from *Play* --> *Play*.
 - For example, in a 4-player game:
   - Red 1 is played on the stacks.
   - It is the *Early Game* and the *Low Score Phase*.
   - Alice clues red to Cathy, touching a new red 2 on slot 2.
   - Cathy hand is, from newest to oldest: `blue 5, red 2, blue 3, blue 2`
   - Bob sees that Alice could have cleanly *5 Pulled* Cathy's red 2. (It is globally known that Alice had another *Play Clue* to give, so a number 5 clue would not have been interpreted as a *5 Stall*.)
   - Bob knows that this signals a blind-play:
-    - A *First Finesse Position* blind-play would look like a *Bluff* on a red 2 or a red 3, which would be a *Lie*.
-    - A *Second Finesse Position* blind-play would look like a *5 Color Ejection* on a red 5, which would be a *Lie*.
-    - A *Third Finesse Position* blind-play would look like an *Unknown Trash Discharge* on a red 1, which would be a *Lie*.
+    - A *First Finesse Position* blind-play would look like a *Bluff* on a red 3, which would be a lie.
```

Comment:
lie should be written as *Lie*

Reply:
yeah didn't mean to change that, weird - one sec